Tools-Override/tests.targets

```diff
@@ -1,21 +1,22 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
+  <!-- This is the target that copies the test assets to the test output -->
+  <Import Project="$(MSBuildThisFileDirectory)publishtest.targets" />
```

Comment:
Is this not imported already? I thought that Build.Common.Targets would import both of these.